tests/test_bundle.py

```diff
@@ -275,7 +275,7 @@ async def test_enable_ingress_auth(ops_test: OpsTest):
     await ops_test.model.wait_for_idle(
         status="active",
         raise_on_blocked=False,
-        timeout=90 * 10,
+        timeout=60 * 20,
```

Comment:
Let's not change the timeout, it was ruled out as not being the cause of failure of the CI, right?

Reply:
That's right, we can keep it. Do you think it's ok if I use `60 * 15`? Same timeout but in more readable format, 15 minutes.

Reply:
Sounds good.